src/network-config.js

```diff
@@ -29,40 +29,60 @@ export const networkConfigs = {
       live: true,
     },
   },
-  [chains.getChainInformation(3).type]: {
-    isActive: false,
+  [chains.getChainInformation(5).type]: {
+    isActive: true,
+    addresses: {
+      ensRegistry:
+        localEnsRegistryAddress || '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
+      dai: DAI_GOERLI_TOKEN_ADDRESS,
+      governExecutorProxy: '0x0451533f685fe028c439821b7502e4cf63b4c32f',
```

Comment:
this address doesn't exist on goerli. https://goerli.etherscan.io/address/0x0451533f685fe028c439821b7502e4cf63b4c32f
Have you deployed it? Because I cannot remember deploying govern to goerli

Reply:
I did not include `Goerli` network. Looking the commit history the change was introduce 2 years ago by Ramon Canales: https://github.com/aragon/client/commit/f400dd951d0e7592a608991cc732d5ab555745a2

Reply:
okey interessting... then lets finally merge